Src/MoneyFox.Windows/App.xaml.cs

```diff
@@ -102,6 +102,7 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
 
                 BackgroundTaskHelper.Register(typeof(ClearPaymentsTask), new TimeTrigger(60, false));
                 BackgroundTaskHelper.Register(typeof(RecurringPaymentTask), new TimeTrigger(60, false));
+				BackgroundTaskHelper.Register(typeof(LiveTiles), new TimeTrigger(15, false));
```

Comment:
Same here, is there a reason for the difference in indent?

Reply:
same

Reply:
resolved in latest updates